synapse/handlers/message.py

```diff
@@ -392,6 +392,13 @@ def assert_accepted_privacy_policy(self, requester):
         if requester.app_service is not None:
             return
 
+        # Check if the user has accepted the privacy policy. We only do this if
```

Comment:
should we not do this by skipping the call to `assert_accepted_privacy_policy`, in the same way that we currently set `ratelimit=False` ?